utils/oomparser/oomparser.go

```diff
@@ -38,6 +38,9 @@ var firstLineRegexp *regexp.Regexp = regexp.MustCompile(
 // struct to hold file from which we obtain OomInstances
 type OomParser struct {
 	systemFile string
+
+	// Inotify event watcher.
+	watcher *inotify.Watcher
```

Comment:
/var/log/messages gets debug logs from many sources. The watcher might get invoked more than once per second.

Were we doing a periodic poll earlier? Can we just check once every 10s and keep an open fd?
(Sorry, I haven't looked at all details of oom-parser. But I feel like watching might not help some of the machines).